src/libraries/Microsoft.Extensions.Caching.Memory/src/Microsoft.Extensions.Caching.Memory.csproj

```diff
@@ -3,6 +3,7 @@
   <PropertyGroup>
     <TargetFrameworks>netstandard2.0;net461</TargetFrameworks>
     <EnableDefaultItems>true</EnableDefaultItems>
+    <NoWarn>$(NoWarn);CP0001</NoWarn>
```

Comment:
Even though we don't have granular suppressions yet, I'd like to see what it is we are suppressing here.

Reply:
> should we add a txt file with the breaking changes ?

Reply:
I don't think we should add a txt file at the moment, that will become descriptive once we have granular suppressions. However it would be good to understand what breaking changes happened as we didn't have this validation of previous vs current for OOB packages. So maybe include a gist or something that we can skim through and see if they are legit?

Reply:
I agree. If the breaking changes aren't many a code comment should be sufficient until the more granular suppression mechanism comes available.